guides/common/modules/proc_adding-an-scc-account.adoc

```diff
@@ -34,3 +37,49 @@ For more information, see xref:Mirroring_Policies_Overview_{context}[].
 Note that you have to re-enter your password if you have already saved your SCC account to {Project}.
 . Click *Submit* to save your SCC account to {Project}.
 . In the {ProjectWebUI}, navigate to *Content > SUSE Subscriptions*, select your SCC account, and click *Sync* to fetch a list of products associated to your SCC account.
+
+[id="cli-Adding_an_SCC_Account_to_Server_{context}"]
+.CLI procedure
+. Optional: Import the public GPG key from SUSE into {Project}.
++
+For more information, see {ContentManagementDocURL}cli-importing-a-gpg-key[Importing a GPG Key] in _{ContentManagementDocTitle}_.
+. Add your SCC account to {Project}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer scc_manager scc_accounts create \
+--base-url "https://scc.suse.com/" \
+--interval _My_Interval_ \
+--katello-gpg-key-id _My_GPG_Key_ID_ \
+--location-id _My_Location_ID_ \
+--login "_My_SCC_Account_Name_" \
+--name "_My_Account_Name_" \
+--organization-id _My_Organization_ID_ \
+--password "_My_SCC_Account_Password" \
+--sync-date _My_Sync_Date_
+----
+. Test your SCC account credentials:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# hammer scc_manager scc_accounts test_connection \
+--id _My_SCC_Account_ID_
+----
++
+Ensure Hammer CLI returns `Testing connection for SCC account succeeded.`.
```

Comment:
```suggestion
Ensure the command returns `Testing connection for SCC account succeeded.`
```
In my view, one dot here is sufficient, however; if you have some guidelines for dealing with this type of sentence please share.

Reply:
Swapped "`" and "." & applied.